pros/cli/upload.py

```diff
@@ -83,6 +83,16 @@ def upload(path: Optional[str], project: Optional[c.Project], port: str, **kwarg
         raise dont_send(click.UsageError('No port provided or located. Make sure to specify --target if needed.'))
 
     if kwargs['target'] == 'v5':
+        slot = kwargs.get('slot', None)
+        if (slot == None):
```

Comment:
we're in python here 😃 
```suggestion
        if slot is not None:
```